src/test/kotlin/org/rust/ide/actions/RsFunctionMoverTest.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.actions
+
+import com.intellij.openapi.actionSystem.IdeActions.ACTION_MOVE_STATEMENT_DOWN_ACTION
+import com.intellij.openapi.actionSystem.IdeActions.ACTION_MOVE_STATEMENT_UP_ACTION
+import org.intellij.lang.annotations.Language
+import org.rust.lang.RsTestBase
+
+class RsFunctionMoverTest : RsTestBase() {
+
+    fun testStepUp() = moveTest("""
+
+            fn <selection>test() {
+                </selection>
+            }
+        """, """
+fn test() {
+
+}
+
+        """, false)
+
+    fun testStepOverFunctionUp() = moveTest("""
+            fn foo() {}
+            fn <selection>test() {
+                </selection>
+            }
+        """, """
+fn test() {
+
+}
+fn foo() {}
+        """, false)
+
+    fun moveTest(
+        @Language("Rust") before: String,
+        @Language("Rust") after: String, down: Boolean) {
```

Comment:
Boolean parameters like `down: Boolean` often are code smell. Let's make two functions instead, `testMoveUp` and `testMoveDown`, to make it easier to find out what is supposed to happen at a glance!